### PR TITLE
:bug: Fix error due to Image.Resampling missing in PIL < 9.1

### PR DESCRIPTION
**Pull request by Maxr1998**

### CODE REVIEW

**Comment by jepler** [comment]
@pepalogik   can you check if this resolves the problem for you?

File: filechooser.py [hunk 1]
```diff
@@ -47,7 +47,11 @@ def update_preview_cb(file_chooser, preview):
         try:
             i = Image.open(filename)
             r = cropgui_common.image_rotation(i)
-            i.thumbnail((PREVIEW_SIZE, PREVIEW_SIZE), Image.Resampling.LANCZOS)
+            if hasattr(Image, 'Resampling'):
+                thumb_filter = Image.Resampling.LANCZOS
+            else:
+                thumb_filter = Image.LANCZOS
+            i.thumbnail((PREVIEW_SIZE, PREVIEW_SIZE), thumb_filter)
```
Comment (pauloxnet) [inline filechooser.py:54]:
```suggestion
            try:
                thumb_filter = Image.Resampling.LANCZOS
            except AttributeError:
                thumb_filter = Image.LANCZOS
            i.thumbnail((PREVIEW_SIZE, PREVIEW_SIZE), thumb_filter)
```
Or something more pythonic?

Comment (Maxr1998) [reply filechooser.py:54]: I'm no Python expert, but after some googling I found another potential solution:

```python
getattr(Image, 'Resampling', Image).LANCZOS
```

This is shorter and also more performant. Lmk what you prefer.

**Comment by jepler** [comment]
Thanks all!

**Comment by Maxr1998** [comment]
Thank you for the merge (and applying the suggestion, didn't come around to do that yesterday).

**Comment by pepalogik** [comment]
It seems everyone thanks everyone :-) . I thank you guys, because you wrote and maintain the program!

**Pull request merged**